### DISCUSSION TITLE
Error in Ross_Report

### DISCUSSION THREAD

**Discussion by kolluruk** [petrobras/ross#1026] [General]
Hi, thank you for making such useful software opensource.

I am trying to run Ross-Report tutorial given at the end of this page as "more complete example"
https://github.com/ross-rotordynamics/ross-report
Tutorial link: https://hub.ovh2.mybinder.org/user/ross-rotordynamics-ross-report-b4xso99c/notebooks/examples/Complete-Example.ipynb

When I run the code I am getting following error:
AttributeError: 'ModalResults' object has no attribute 'calc_mode_shape'
with reference to the following code line in analysis.py in report folder
xn, yn, zn, xc, yc, zc_pos, nn = modal.calc_mode_shape(mode=mode)

Can you suggest a solution for this?

Thank you.

**Comment by raphaeltimbo** [comment]
Hi, @kolluruk. 
Unfortunately the ross-report repository is not well maintained and is using obsolete code.
We are evaluating archiving that repository and creating a report option in the official ross repository by means of a jupyter-book application.
